src/components/Header/Header.js

```diff
@@ -45,23 +45,37 @@ function Header() {
     return (
         <>
             <header className={styles.Header}>
+              <div className="container mx-auto">
+               <div className="sm:flex justify-around">
                 <img className={styles.left} src={logo} alt="Logo"/>
-                <div className={styles.right}>
-                    <Link to='home_banner' offset={-15}>
+                  <ul className="text-gray-700 sm:self-center text-xl border-t sm:border-none">
+                    <li className="sm:inline-block">
+                    <Link to='home_banner' >
                         <NavLink className={styles.header_links} to='/' exact activeClassName={styles.header_active_links}>
                             Home
                         </NavLink>
                     </Link>
+                    </li>
+                    <li className="sm:inline-block">
                     <NavLink className={styles.header_links} to='/editor' exact activeClassName={styles.header_active_links}>
                         Editor
                     </NavLink>
+
+                    </li>
+                    <li className="sm:inline-block">
+                    <NavLink className={styles.header_links} to='/sketch' exact activeClassName={styles.header_active_links}>
+                        Sketch <sup><span style={{color: "red", fontSize: "0.7rem"}}>New</span></sup>
```

Comment:
@Amit366  remove the code in line 66 and 67. that code is causing the build error.